src/bootstrap/compile.rs

```diff
@@ -177,7 +177,7 @@ fn copy_third_party_objects(
     }
 
     if target == "x86_64-fortanix-unknown-sgx"
-        || builder.config.llvm_libunwind == LlvmLibunwind::InTree
+        || builder.config.target_config[&target].llvm_libunwind == LlvmLibunwind::InTree
```

Comment:
If you grep for other usages of target_config you'll see we typically do a get(...) rather than direct indexing, because it's fairly common for no target-specific configuration to be present and so there's no entry in the table.

(It may be good to generate defaults in config.rs and avoid this, but that's a larger change.)